packages/webdriver-utils/src/metadata/commonMetaDataResolver.js

```diff
@@ -0,0 +1,14 @@
+import CommonDesktopMetaDataResolver from './commonDesktopMetaDataResolver.js';
+import CommonMobileMetaDataResolver from './commonMobileMetaDetaResolver.js';
+
+export default class CommonMetaDataResolver {
+  static resolve(driver, capabilities, opts) {
+    if (!driver) throw new Error('Please pass a Driver object');
```

Comment:
```suggestion
    if (!driver) throw new Error('Please pass a Driver object');

```
add empty line after guard conditions